docs/sdk/tutorials/export_a_kili_project.md

```diff
@@ -0,0 +1,207 @@
+| YOLO V4       | ✅   | ✅             | ✅                      |
+| YOLO V5       | ✅   | ✅             | ✅                      |
+| YOLO V7       | ❌   | ✅             | ✅                      |
+| Pascal VOC    | ✅   | ❌             | ❌                      |
+| COCO          | ❌   | ✅             | ✅                      |
+
+
+### The `.export_labels` method
+
+The `.export_labels` method enables the export of a full project. It does the following preprocessing:
+
+* Only fetches the labels of types `"DEFAULT"` and `"REVIEW"`.
+* If specified, only selects a subset of asset ids.
+* You can specify a standard export format (only available for a restricted set of ML task).
+* Depending on the export type, it downloads the asset media:
+     * Kili: ❌
+     * YOLO: ❌
+     * COCO: ✅
+* Depending on the `export_type` argument, it will export the latest label or all the labels.
+* Depending on the `split_option` argument, the layout will be one folder for all the jobs, or one folder per job.
+* The `single_file` argument tells if the labels data should be exported into one single file. Note that some formats are single_file only, and that some others cannot be output into a single file.
+     * Kili: single file or multiple files.
+     * YOLO: multiple files only.
+     * COCO: single-file only.
+
+For all the formats, in the output archive, a README.kili.txt file is also output. Here is an example of its contents:
+```
+Exported Labels from KILI
+=========================
+
+- Project name: Awesome annotation project
+- Project identifier: abcdefghijklmnop
+- Project description: This project contains labels, most of which are awesome.
+- Export date: 20221125-093324
+- Exported format: kili
+- Exported labels: latest
+```
+
+
+### Kili format, one file per asset
+```python
+from kili.client import Kili
+kili = Kili()
+kili.export_labels(
+    project_id = "<your_project_id>",
+    filename = "/tmp/export.zip",
+    fmt = "kili",
+)
+```
+will export all the assets payloads and the associated labels, with one json file per asset, into the `/tmp/export.zip` folder.
+
+### Kili format, one file for the whole project
+```python
+from kili.client import Kili
+kili = Kili()
+kili.export_labels(
+    project_id = "<your_project_id>",
+    filename = "/tmp/export.zip",
+    fmt = "kili",
+    single_file = True,
+)
+```
+will export all the assets payloads and the associated labels, with one json file per asset, into the `/tmp/export.zip` folder.
+
+### YOLO formats
+You can also export to the yolo format, when you have at least one Object Detection job with bounding boxes. You can select the `"yolo_v4"`, `"yolo_v5"` or `"yolo_v7"`. The difference between each format is the structure of the metadata YAML file, which specifies the object classes. In all the cases, it produces one file per asset in the Yolo format, containing the last DEFAULT or REVIEW label that has been produced. Each YOLO label has the following shape:
```

Comment:
"you can select"
comment: you don't really select. I think we need to phrase it differently, I think. Maybe "choose"?